### PR TITLE
Custom drag layer

### PR DESCRIPTION
**Pull request by mattkrick**
After moving back to react-dnd, we lost the full opacity of the card being dragged.
This restores the full opacity.

We don't use react-dnd to manage drag state. That's because we'll want to share this state with other clients, so we have to manage it on our own. 

TEST
- [x] can drag a reflection & the reflection being dragged is at full opacity, without footers or special stuff.

### CODE REVIEW

File: src/universal/components/ReflectionCardInFlight.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import ReflectionEditorWrapper from 'universal/components/ReflectionEditorWrapper';
+import {ReflectionCardRoot} from 'universal/components/ReflectionCard/ReflectionCard';
+import {convertFromRaw, EditorState} from 'draft-js';
+import styled from 'react-emotion';
+import ui from 'universal/styles/ui';
+import withAtmosphere from 'universal/decorators/withAtmosphere/withAtmosphere';
+import {commitLocalUpdate, createFragmentContainer} from 'react-relay';
+import type {ReflectionCardInFlight_reflection as Reflection} from './__generated__/ReflectionCardInFlight_reflection.graphql';
+
+type Coords = {
```
Comment (jordanh) [inline src/universal/components/ReflectionCardInFlight.js]: Look at you with your fancy typing

**Comment by jordanh** [comment]
# Test Cycle
- [x] can drag a reflection & the reflection being dragged is at full opacity, without footers or special stuff.
- [x] can ungroup

**Comment by jordanh** [comment]
@mattkrick this looks very good. I trust it'll still work if you resolve the comments? If so, please merge. It'd be good to work toward getting a hotfix out tonight/tomorrow for folks early beta folks who might be running a retro

**Comment by mattkrick** [comment]
merged to staging as a hotfix

**Pull request merged**